### DISCUSSION TITLE
Linux precompiled binaries

### DISCUSSION THREAD

**Discussion by boimebenoit** [FlareSolverr/FlareSolverr#742] [Q&A]
Hi ! Is it normal that the precompiled binary for linux is not a zip file like the old ones ? I can't unzip it even by renaming the file to .zip.

![image](https://user-images.githubusercontent.com/71324635/226739403-84323d41-05d7-4082-9769-cc5cfe9ebe8e.png)

**Comment by ngosang** [comment]
It's an executable.
```
chmod +x flaresolverr_linux_x64
./flaresolverr_linux_x64
```
I will provide a `.zip` as part of the fix for this issue:  #741

**Reply by boimebenoit** [reply]
Thanks !
